test/driver.hpp

```diff
@@ -56,6 +56,23 @@ constexpr std::is_same<T, U> is_same(const T&)
     return {};
 }
 
+struct tensor_elem_gen_integer
```

Comment:
Please check if this change should come from your PR

Reply:
This struct is defined here on develop branch
test/tensor_holder.hpp